### DISCUSSION TITLE
Whitebeet - Looking for a Command that doesn't generate a response frame

### DISCUSSION THREAD

**Discussion by Bobbybigears** [Sevenstax/FreeV2G#129] [Q&A]
Hi, 

I'm developing a prototype that uses a C-based SPI host. I'm looking for a command that I can send which won't generate a response frame from the Whitebeet. In essence, I need a command which will remove frame(s) from the Whitebeet's TX Queue without adding frames to the queue as well.
Thanks in advance
Rob

**Comment by jpo-stx** [comment]
You don't need to send a command. If the TX_PENDING pin is high just read out the frame by sending a size of zero in your size exchange frame and read out the number of bytes the whitebeet is sending in its size exchange frame.
